src/components/QuizGame/index.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import { fetchQuizQuestions } from "../../components/QuizGameApi";
+import QuestionCard from "../../components/QuizGameQuestions";
+import "./index.css";
+
+const DIFFICULTY = {
+  EASY: "easy",
+  MEDIUM: "medium",
+  HARD: "hard",
+};
+const TOTAL_QUESTIONS = 10;
+const QuizGame = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [questions, setQuestions] = useState([]);
+  const [currentQuestionNumber, setCurrentQuestionNumber] = useState(0);
+  const [userAnswers, setUserAnswers] = useState([]);
+  const [score, setScore] = useState(0);
+  const [isGameStart,setIsGameStart]=useState(false)
+  const startQuiz = async () => {
```

Comment:
newline above. in general use more newlines especially between long function declaration, please fix throughout whole file